js/Common.js

```diff
@@ -1323,6 +1323,7 @@ if (!CRM.vars) CRM.vars = {};
         ele.one('change', function () {
           if (msg && msg.close) msg.close();
           ele.removeClass('error');
+          ele.removeClass('crm-error');
```

Comment:
This works to remove the redness when I test it but wondering where is the plain 'error' one added?

(And if you want be extra cool you could chain, e.g. `ele.removeClass('error').removeClass('crm-error');`

FYI I see a pre-existing issue where if the title is passed in I got an error because label right below this then ends up undefined.

Reply:
@demeritcowboy I think that the error class is coming from jquery validation I think

Reply:
See my other comment here. `error` is a bootstrap class and is not being set by the CiviCRM jquery validate implementation (we are using `crm-inline-error` there).